modules/picard/sortvcf/meta.yml

```diff
@@ -0,0 +1,40 @@
+name: picard_sortvcf
+description: Sorts vcf files
+keywords:
+  - sort
+  - vcf
+tools:
+  - picard:
+      description: Java tools for working with NGS data in the BAM/CRAM/SAM and VCF format
+      homepage: https://broadinstitute.github.io/picard/
+      documentation: https://broadinstitute.github.io/picard/
```

Comment:
recommend this specific [link](https://broadinstitute.github.io/picard/command-line-overview.html#SortVcf) for documentation

Reply:
Fixed 👍🏻